packages/velocity-exchange-verifiers/src/impl/verifiers/pure-verifiers/iss-claim-matches-either-metadata-or-credential-issuer-verifier.ts

```diff
@@ -35,12 +35,12 @@ import { buildError, ERROR_CODES } from 'impl/errors';
  * @validationRule `credential.payload.iss` must equal `credential_issuer_metadata.iss` or `credential_issuer_metadata.credential_issuer`.
  * @errorCode `UNEXPECTED_CREDENTIAL_PAYLOAD_ISS` — if the issuer does not match either expected value.
  *
- * @see {@link CredentialJwt}
+ * @see {@link W3CCredentialJwtV1}
  * @see {@link VerificationError}
  * @see {@link VerificationContext}
  */
 export const issClaimMatchesEitherMetadataOrCredentialIssuerVerifier: Verifier<
-  CredentialJwt
+  W3CCredentialJwtV1
 > = (credential, context) => {
   const actual = credential.payload?.iss;
   const allowed = [
```

Comment:
- this code should assume that context.credential_issuer_metadata is NOT NULL
- If `iss` or `credential_issuer` is null, then it IS NOT an `allowed` value so should be removed. 
- also `payload` cannot be null, nor can `payload.iss`